spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/ConfigMapsTest.java

```diff
@@ -18,64 +18,58 @@
 
 import java.util.Map;
 
+import io.fabric8.kubernetes.api.model.ConfigMap;
 import io.fabric8.kubernetes.api.model.ConfigMapBuilder;
 import io.fabric8.kubernetes.api.model.ConfigMapList;
-import io.fabric8.kubernetes.api.model.ConfigMapListBuilder;
 import io.fabric8.kubernetes.client.KubernetesClient;
-import io.fabric8.kubernetes.client.server.mock.KubernetesServer;
-import org.junit.Rule;
-import org.junit.Test;
+import io.fabric8.kubernetes.client.server.mock.EnableKubernetesMockClient;
+import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.springframework.cloud.kubernetes.fabric8.config.ConfigMapTestUtil.readResourceFile;
 
 /**
  * @author Charles Moulliard
  */
+@EnableKubernetesMockClient(crud = true, https = false)
 public class ConfigMapsTest {
 
-	@Rule
-	public KubernetesServer server = new KubernetesServer();
+	private static KubernetesClient mockClient;
 
 	@Test
 	public void testConfigMapList() {
-		this.server.expect().withPath("/api/v1/namespaces/ns1/configmaps")
-				.andReturn(200, new ConfigMapListBuilder().build()).once();
+		mockClient.configMaps().inNamespace("ns1").createNew();
 
-		KubernetesClient client = this.server.getClient();
-
-		ConfigMapList configMapList = client.configMaps().inNamespace("ns1").list();
+		ConfigMapList configMapList = mockClient.configMaps().inNamespace("ns1").list();
 		assertThat(configMapList).isNotNull();
 		assertThat(configMapList.getItems().size()).isEqualTo(0);
 	}
 
 	@Test
 	public void testConfigMapGet() {
-		this.server
-				.expect().withPath("/api/v1/namespaces/ns2/configmaps").andReturn(200, new ConfigMapBuilder()
-						.withNewMetadata().withName("reload-example").endMetadata().addToData("KEY", "123").build())
-				.once();
 
-		KubernetesClient client = this.server.getClient();
-		ConfigMapList configMapList = client.configMaps().inNamespace("ns2").list();
+		ConfigMap configMap = new ConfigMapBuilder().withNewMetadata().withName("reload-example").endMetadata()
+				.addToData("KEY", "123").build();
+
+		mockClient.configMaps().inNamespace("ns2").create(configMap);
+
+		ConfigMapList configMapList = mockClient.configMaps().inNamespace("ns2").list();
 		assertThat(configMapList).isNotNull();
-		assertThat(configMapList.getAdditionalProperties()).containsKey("data");
-		@SuppressWarnings("unchecked")
-		Map<String, String> data = (Map<String, String>) configMapList.getAdditionalProperties().get("data");
-		assertThat(data.get("KEY")).isEqualTo("123");
+		assertThat(configMapList.getItems().size()).isEqualTo(1);
```

Comment:
it tests the same thing, but where to find the "thing" to assert against is in a different place